controller/mariadb_controller_test.go

```diff
@@ -130,33 +130,33 @@ var _ = Describe("MariaDB controller", func() {
 			}, testTimeout, testInterval).Should(BeTrue())
 
 			By("Expecting to create a exporter Deployment eventually")
-			Eventually(func() bool {
+			Eventually(func(g Gomega) bool {
 				var deploy appsv1.Deployment
 				if err := k8sClient.Get(testCtx, testMariaDb.MetricsKey(), &deploy); err != nil {
 					return false
 				}
 				expectedImage := os.Getenv("RELATED_IMAGE_EXPOTER")
-				Expect(expectedImage).ToNot((BeEmpty()))
+				g.Expect(expectedImage).ToNot(BeEmpty())
 				By("Expecting Deployment to have exporter image")
-				Expect(deploy.Spec.Template.Spec.Containers).To(ContainElement(MatchFields(IgnoreExtras,
+				g.Expect(deploy.Spec.Template.Spec.Containers).To(ContainElement(MatchFields(IgnoreExtras,
 					Fields{
 						"Image": Equal(expectedImage),
 					})))
 				return deploymentReady(&deploy)
-			})
+			}).Should(BeTrue())
 
 			By("Expecting to create a ServiceMonitor eventually")
-			Eventually(func() bool {
+			Eventually(func(g Gomega) bool {
 				var svcMonitor monitoringv1.ServiceMonitor
 				if err := k8sClient.Get(testCtx, testMariaDb.MetricsKey(), &svcMonitor); err != nil {
 					return false
 				}
-				Expect(svcMonitor.Spec.Selector).NotTo(BeNil())
-				Expect(svcMonitor.Spec.Selector).To(HaveKeyWithValue("app.kubernetes.io/name", "mariadb"))
-				Expect(svcMonitor.Spec.Selector).To(HaveKeyWithValue("app.kubernetes.io/instance", testMariaDbName))
-				Expect(svcMonitor.Spec.Endpoints).To(HaveLen(1))
+				g.Expect(svcMonitor.Spec.Selector).NotTo(BeNil())
+				g.Expect(svcMonitor.Spec.Selector).To(HaveKeyWithValue("app.kubernetes.io/name", "mariadb"))
+				g.Expect(svcMonitor.Spec.Selector).To(HaveKeyWithValue("app.kubernetes.io/instance", testMariaDbName))
+				g.Expect(svcMonitor.Spec.Endpoints).To(HaveLen(1))
 				return true
-			})
+			}).Should(BeTrue())
```

Comment:
```suggestion
			}, testTimeout, testInterval).Should(BeTrue())
```
Same as previous block.